src/neo/Ledger/Blockchain.cs

```diff
@@ -62,7 +62,6 @@ public class RelayResult { public IInventory Inventory; public VerifyResult Resu
         private readonly NeoSystem system;
         private readonly List<UInt256> header_index = new List<UInt256>();
         private uint stored_header_count = 0;
-        private readonly Dictionary<UInt256, Block> block_cache = new Dictionary<UInt256, Block>();
```

Comment:
Maybe the `block_cache` should be kept? #1592

Reply:
Yes, I kept it and test now.